frontend/src/components/layout/PageContainer.tsx

```diff
@@ -85,11 +86,13 @@ export const PageContainer: FC<Props> = ({
           {children}
         </QueryResultsPageContent>
       ) : (
-        <Container component="main" maxWidth={maxWidth} sx={containerStyles}>
-          {children}
-        </Container>
+        <>
+          <Container component="main" maxWidth={maxWidth} sx={containerStyles}>
+            {children}
+          </Container>
+          {isLoading ? <LinearProgress /> : <Box sx={{ height: 4 }} />}
+        </>
       )}
-      {isLoading ? <LinearProgress /> : <Box sx={{ height: 4 }} />}
```

Comment:
this loader wil now not be displayed for the query results page, just noting in case it wasn't intentional

Reply:
Yes, was intentional. From my POV it's just OTT to have a spinner and a progress bar. Probably a bit more sensitive to this as I find screen motion difficult. 

It's not for now, but as a more general point I think it would be good to make sure that we honour reduced motion preferences (the last time I tried to confirm that MUI supports this, I couldn't definitively clarify).

Reply:
Yeah, I think it should be automatically handled by MUI. Not saying that it is, but it should be :)